apps_download/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "Product Download for Appstore",
+    "version": "10.0.0.0.1",
+    "author": "BizzAppDev, AgentERP, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/apps-store",
+    "license": "AGPL-3",
+    "category": "Uncategorized",
```

Comment:
Category as eCommerce should do.

Reply:
I do not think so as for eCommerce we will have another module for that which will inherit this module and takes care for the views of eCommerce.

Reply:
then, sale will be right.